studio/pages/project/[ref]/auth/settings.tsx

```diff
@@ -331,7 +341,38 @@ const Settings = () => {
                       errorMessage="Please enter the twilio message service sid."
                     />
                   </>
-                )}
+                ) : (smsProviderModel?.SMS_PROVIDER === 'vonage' ? (
+                  <>
+                    <AutoField
+                      name="SMS_VONAGE_API_KEY"
+                      showInlineError
+                      errorMessage="Please enter the vonage account sid."
+                    />
+                    <SecretField
+                      name="SMS_VONAGE_API_SECRET"
+                      showInlineError
+                      errorMessage="Please enter the vonage auth token."
+                    />
+                    <AutoField
+                      name="SMS_VONAGE_FROM"
+                      showInlineError
+                      errorMessage="Please enter the vonage message service sid."
+                    />
+                  </>
+                ) : (
+                  <>
+                    <AutoField
+                      name="SMS_TEXTLOCAL_API_KEY"
+                      showInlineError
+                      errorMessage="Please enter the vonage account sid."
+                    />
+                    <SecretField
+                      name="SMS_TEXTLOCAL_SENDER"
+                      showInlineError
+                      errorMessage="Please enter the vonage auth token."
+                    />
+                  </>
+                )))}
```

Comment:
this makes the default phone provider "textlocal", can we leave the default to use twilio instead?